moshi/src/main/java/com/squareup/moshi/JsonAdapter.java

```diff
@@ -223,12 +240,19 @@ public final void toJson(BufferedSink sink, @Nullable T value) throws IOExceptio
           writer.setIndent(originalIndent);
         }
       }
+      @Override boolean isLenient() {
+        return delegate.isLenient();
+      }
       @Override public String toString() {
         return delegate + ".indent(\"" + indent + "\")";
       }
     };
   }
 
+  boolean isLenient() {
```

Comment:
I did this with a field for fewer lines of code, but changed it to a method in 4c4c71910381f9dad7871475cc7cbb0c1e95468a because it seems clearer than using a constructor parameter.